packages/eslint-config-base/style.js

```diff
@@ -443,24 +444,21 @@ module.exports = {
     // Require or disallow padding lines between statements
     // This rule requires or disallows blank lines between the given 2 kinds of statements. Properly
     // blank lines help developers to understand the code.
-    'padding-line-between-statements': ['warn',
-      // Newline after directives ("use strict" etc.), but not between them
-      { blankLine: 'always', prev: 'directive', next: '*' },
+    'padding-line-between-statements': [
```

Comment:
This obviously-prettier-made formatting makes me want to puke but... otherwise ok.